sources/packages/api/src/services/student-appeal/student-appeal.service.ts

```diff
@@ -335,4 +338,92 @@ export class StudentAppealService extends RecordDataModelService<StudentAppeal>
         .save(appealToUpdate);
     });
   }
+
+  /**
+   * Get all pending student appeals.
+   * @param paginationOptions options to execute the pagination.
+   * @returns StudentAppeal list.
+   */
+  async getAppealsByStatus(
+    paginationOptions: PaginationOptions,
+  ): Promise<PaginatedResults<StudentAppeal>> {
+    const studentAppealsQuery = this.repo
+      .createQueryBuilder("studentAppeal")
+      .select([
+        "studentAppeal.id",
+        "application.id",
+        "studentAppeal.submittedDate",
+        "user.firstName",
+        "user.lastName",
+        "application.applicationNumber",
+        "student.id",
+      ])
+      .innerJoin("studentAppeal.application", "application")
+      .innerJoin("application.student", "student")
+      .innerJoin("student.user", "user")
+      .innerJoin("studentAppeal.appealRequests", "appealRequests")
+      .where(
```

Comment:
Can't we use a where here without using exists? 
like 
.where("appealRequests.appealStatus = :appealStatus" , {appealStatus: StudentAppealStatus.Pending})

Correct me if I am wrong.

Reply:
Doing that will return the same appeal multiple times in the summary table.

Reply:
Ah I see. Nevermind. I was in an understanding that we are trying to show all appeal requests here.